core/plugins/org.polarsys.capella.core.sirius.analysis/description/common.odesign

```diff
@@ -2909,8 +2909,272 @@
         </edgeMappings>
         <edgeMappings name="DT_TitleBlockEdge" targetMapping="//@ownedViewpoints[name='Common']/@ownedRepresentations[name='Class%20Diagram%20Blank']/@defaultLayer/@containerMappings[name='DT_TitleBlockContainer']" targetFinderExpression="service:computeTitleBlockElements(diagram)">
           <sourceMapping xsi:type="description_1:ContainerMapping" href="#//@ownedViewpoints[name='Common']/@ownedRepresentations[name='Class%20Diagram%20Blank']/@defaultLayer/@containerMappings[name='DT_Class']"/>
+          <sourceMapping xsi:type="description_1:ContainerMapping" href="#//@ownedViewpoints[name='Common']/@ownedRepresentations[name='Class%20Diagram%20Blank']/@defaultLayer/@containerMappings[name='DT_Collection']"/>

```

Comment:
Could you look at how the constraint edge is implemented? It does not add all the possible source/target mapping.

Reply:
Oops. Sorry that's normal when you import the tool into other diagrams, these sourceMappings are added automatically.